### ISSUE TITLE
Adding Mailinglist Inxmail

### ISSUE THREAD

**Issue by CKa0121** [rspamd/rspamd#4751]
## Summary

Kindly asking to enhance the maillist module to cover also the commercial mail list provider Inxmail with their mailer of the same name Inxmail.

## Motivation

I hope mailings from this provider (for example a mobile phone manufacturer (one of the big 5 world wide) sending out his announcements) to be recognized by FORGED_SENDER_MAILLIST

## Describe alternatives you've considered

The only alternatives coming to my mind in this case were (a) setting FORGED_SENDER to 0.00 (what I do not like and think it is not the expected way) or (b) adding the sender to my whitelist (but this should in my opinion be only the ultima ratio, works here as a workaround, but as the emails from this service could be easily identified there should be a more generic way).

## Additional context

Inxmail includes following headers:

List-Id: "0" <SOME-UID>
List-Unsubscribe: <https://URL>, <mailto:EMAIL-RECIPIENT?subject=XYZ>
List-Unsubscribe-Post: List-Unsubscribe=One-Click
X-Mailer: Inxmail EE 4.8.56.127

Optionally (with the list maintainer I just now checked it exists):
Feedback-ID: XYZ:XYZ:inxmailde

**Comment by fatalbanana** [comment]
Do you find this sender gets classified as spam?- `FORGED_SENDER` is only worth 0.3 points.

**Comment by CKa0121** [comment]
Yes. Nowadays I have a quite well set configuration (fitting almost all email I got here). So I set a tight score to classify as spam vs. ham. For this 0.3 could do the difference.

On the other hand: if there is the occassion to distinguish between 'real' FORGED_SENDER and FORGED_SENDER_MAILLIST - why not make use of it?

**Comment by fatalbanana** [comment]
I get impression `MAILLIST` is rather intended to detect things more akin to discussion groups. If it wouldn't be the right place for it we might add some rules aimed more at generic bulk/transactional sorts of mails and/or identifying specific ESPs.

I wonder what else is driving up score that trying to keep it down is a concern.
